requirements.txt

```diff
@@ -1,6 +1,6 @@
 # BioSimSpace runtime requirements.
 
-sire ~=2023.0.0   # this matches >= 2023.0.0 && < 2023.1.0
+sire ~=2023.0.1   # this matches >= 2023.0.1 && < 2023.1.0
```

Comment:
Are expecting some change for 2023.1.0?

Reply:
Potentially. 2023 is currently pre-release so the API hasn't been nailed down yet. I don't expect anything to change in a breaking way, but we don't want these pre-release builds to continue building against the full release. At that point, we'll have a 2023.1.0 release and will update the requirements accordingly.